.travis.yml

```diff
@@ -24,3 +24,5 @@ after_success:
   - bin/createcoverage
   - pip install coverage==3.7.1 coveralls
   - coveralls
+  - docker.io build --tag plone.restapi:$(git rev-parse HEAD) .
+#  - docker.io push kitconcept/plone.restapi
```

Comment:
I am a bit confused. On Ubuntu I need to user docker.io. Maybe travis is using a symlink to be able to use docker instead when you use a docker service. Anyways, will test this as soon as I have the plone dockerhub credentials. :)